src/main/kotlin/org/malachite/estella/mails/Payloads.kt

```diff
@@ -71,4 +73,13 @@ fun userRegistrationMailPayload(user: User) =
         sender_email = MAIN_MAIL
     )
 
+fun hrPartnerRegistrationMailPayload(hrpPartner: HrPartner, password: String) =
```

Comment:
Ten hrpPartner to chyba missclick słowny
```suggestion
fun hrPartnerRegistrationMailPayload(hrPartner: HrPartner, password: String) =
```